testing/Dockerfile.bot

```diff
@@ -1,4 +1,7 @@
 FROM rust
-RUN apt-get update && apt-get install -y build-essential cmake libssh2-1-dev
+RUN apt-get update && \
+    apt-get install -y build-essential cmake libssh2-1-dev && \
+    rm -rf /var/lib/apt/lists/*
 RUN cargo install cargo-watch
-CMD ["/bin/bash", "/src/testing/bot.sh"]
\ No newline at end of file
+WORKDIR /src
+CMD cargo watch -s 'cargo run -- --config testing/config.yml'
```

Comment:
The idea behind the bot.sh script was to enable adding steps to the script and testing them without having to rebuild the docker image.

Reply:
Then you can just use `docker-compose exec` to run a command in the container or `docker-compose run bot [cmd]`, I always try to avoid wrappers.